src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,37 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  constructor(name, health = 100, hidden = false) {
```

Comment:
The 'hidden' property is being initialized in the constructor of the Animal class, but according to the task, only instances of Herbivore should have the 'hidden' property. You should remove the 'hidden' property initialization from the Animal class.